client/datalake/tests/conftest.py

```diff
@@ -19,16 +19,11 @@
 import six
 
 
-try:
-    from moto import mock_s3
-    import boto.s3
-    from boto.s3.key import Key
-    from six.moves.urllib.parse import urlparse
-    import simplejson as json
-except ImportError:
-    # if developers use s3-test features without having installed s3 stuff,
-    # things will fail. So it goes.
-    pass
+from moto import mock_s3_deprecated
+import boto.s3
+from boto.s3.key import Key
+from six.moves.urllib.parse import urlparse
+import json
```

Comment:
Can we put this back in the `try/except`? Some of the test fixtures (e.g., `random_metadata`) which do not require s3 and boto are used by some users. And we don't want to fail them.

Reply:
Done!